vibe_code_apps_20/wattlab/studio/pages/uploads.py

```
@@ -0,0 +1,151 @@
+                    else:
+                        pkg = load_energy_use_package(p)
+                        st.session_state["studio_energy_path"] = str(p)
+                else:
+                    st.warning("Upload an energy zip or enter a path.")
+                    return
+                st.session_state["studio_energy"] = pkg
+                if pkg.campus is not None:
+                    st.session_state["studio_campus"] = pkg.campus
+                    st.session_state["fuel_weather_campus"] = pkg.campus
+                st.success(
+                    f"Energy package loaded"
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)